### PR TITLE
Added config steps for Unraid

### PR DESCRIPTION
**Pull request by magnusoverli**
Added Unraid under hypervisor specific configuration steps

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
     More details can be found about the command can be found [here](https://www.virtualbox.org/manual/ch08.html#vboxmanage-storageattach).
 
     </div>
+
+- title: Unraid
```
Comment (frenck) [inline source/_includes/installation/operating_system.md:280]: This has been scoped by an architectural decision, see:

<https://github.com/home-assistant/architecture/blob/master/adr/0015-home-assistant-os.md>

We don't want to support/add more than decided in there. If we want to extend this list, we need to discuss this in an architectural discussion and approve this first, before we add this to the documentation.

Feel free to open up an architectural proposal in our architecture repo.

../Frenck

**Comment by magnusoverli** [comment]
I was reading there and decided to add this to the docs based on:

> We will provide documentation for the following systems build on top of these technologies:
   > Proxmox (KVM/QEMU based)
   > Unraid (KVM/QEMU based)
   > ...

What am I missing?

File: source/_includes/installation/operating_system.md (hunk 1 above)
Comment (magnusoverli) [reply source/_includes/installation/operating_system.md:280]: I was reading there and decided to add this to the docs based on:

> We will provide documentation for the following systems build on top of these technologies:
>  Proxmox (KVM/QEMU based)
>  Unraid (KVM/QEMU based)
>  ...

What am I missing?

Comment (frenck) [reply source/_includes/installation/operating_system.md:280]: You? Nothing. Me? A lot apparently 🙈

File: source/_includes/installation/operating_system.md [hunk 2]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+  content: |
+    1. Download the **.qcow2** image above and decompress it. (**Extract all** in Windows)
+    2. Store the image in the **isos** share on your server.
+    3. Make sure "Settings" -> "VM manager" -> "Enable VMs" is enabled.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    3. Make sure **Settings** > **VM manager** > **Enable VMs** is enabled.
```

File: source/_includes/installation/operating_system.md [hunk 3]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    4. Create a new virtual machine: VMS -> ADD VM.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    4. Create a new virtual machine: **VMS** > **Add VM**.
```

File: source/_includes/installation/operating_system.md [hunk 4]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    5. Select type **Linux** and give the VM a name and a description.
+    6. Select the CPU cores you want to let the VM use and give it some memory. 
+    7. Under **Primary vDisk Location** select *Manual* and then select the qcow2 image.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    7. Under **Primary vDisk Location**, select **Manual** and then select the qcow2 image.
```

File: source/_includes/installation/operating_system.md [hunk 5]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    8. Select your keyboard language under **VM Console Keyboard**.
+    9. Select *br0* under **Network Source**.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    9. Select **br0** under **Network Source**.
```

File: source/_includes/installation/operating_system.md (hunk 1 above)
Comment (magnusoverli) [reply source/_includes/installation/operating_system.md:280]: Hehe… Super! 👍🏻
Is it too much to ask to get the hacktoberfest label back also? 🫣

File: source/_includes/installation/operating_system.md [hunk 6]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    10. Select *virtio* under **Network model**.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    10. Select **virtio** under **Network model**.
```

File: source/_includes/installation/operating_system.md [hunk 7]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    11. Select any USB-devices that you want to pass through to Home Assistant, such as Zigbee- or Zwave-controllers.
+    12. **Deselect** "Start VM after creation".
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    12. Deselect **Start VM after creation**.
```

File: source/_includes/installation/operating_system.md [hunk 8]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    13. Click **CREATE**.
+    14. Click on **the name** of your new VM and click on the capacity number for your disk. Here you can expand the disk to whatever your needs are. Default 32GB.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    14. Select the name of your new VM and select the capacity number for your disk. Here, you can expand the disk to whatever your needs are. The default is 32&nbsp;GB.
```

File: source/_includes/installation/operating_system.md [hunk 9]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    15. Click on **the icon** of your new VM and select *start with console(VNC)*.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    15. Select the icon of your new VM and select **start with console (VNC)**.
```

File: source/_includes/installation/operating_system.md [hunk 10]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    2. Store the image in the **isos** share on your server.
+    3. Make sure "Settings" -> "VM manager" -> "Enable VMs" is enabled.
+    4. Create a new virtual machine: VMS -> ADD VM.
+    5. Select type **Linux** and give the VM a name and a description.
+    6. Select the CPU cores you want to let the VM use and give it some memory. 
+    7. Under **Primary vDisk Location** select *Manual* and then select the qcow2 image.
+    8. Select your keyboard language under **VM Console Keyboard**.
+    9. Select *br0* under **Network Source**.
+    10. Select *virtio* under **Network model**.
+    11. Select any USB-devices that you want to pass through to Home Assistant, such as Zigbee- or Zwave-controllers.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    11. Select any USB-devices that you want to pass through to Home Assistant, such as Zigbee- or Z-Wave controllers.
```

File: source/_includes/installation/operating_system.md [hunk 11]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
...
+    4. Create a new virtual machine: **VMS** > **Add VM**.
+    5. Select type **Linux** and give the VM a name and a description.
+    6. Select the CPU cores you want to let the VM use and give it some memory. 
+    7. Under **Primary vDisk Location**, select **Manual** and then select the qcow2 image.
+    8. Select your keyboard language under **VM Console Keyboard**.
+    9. Select **br0** under **Network Source**.
+    10. Select **virtio** under **Network model**.
+    11. Select any USB-devices that you want to pass through to Home Assistant, such as Zigbee- or Z-Wave controllers.
+    12. Deselect **Start VM after creation**.
+    13. Click **CREATE**.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    13. Select **Create**.
```

File: source/_includes/installation/operating_system.md [hunk 12]
```diff
@@ -276,6 +276,26 @@ _All these can be extended if your usage calls for more resources._
     More details can be found about the command can be found [here](https://www.virtualbox.org/manual/ch08.html#vboxmanage-storageattach).
 
     </div>
+
+- title: Unraid
+  content: |
+    1. Download the **.qcow2** image above and decompress it. (**Extract all** in Windows)
+    2. Store the image in the **isos** share on your server.
+    3. Make sure **Settings** > **VM manager** > **Enable VMs** is enabled.
```
Comment (c0ffeeca7) [inline source/_includes/installation/operating_system.md]:
```suggestion
    3. Make sure under **Settings** > **VM manager**, **Enable VMs** is enabled.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @magnusoverli :+1:

**Pull request merged**